jobs/cloud_controller_ng/spec

```diff
@@ -253,6 +253,10 @@ consumes:
   optional: true
 
 properties:
+  set_ulimit_fds:
```

Comment:
likewise, please namespace this under `cc.set_ulimit_fds` and move it down

Reply:
Done with commit 3bc3b594c92e6f3b054ea16dd1060593ee7020f1.